### PR TITLE
Extension npm packaging preparations

### PR DESCRIPTION
**Pull request by jakolehm**
Signed-off-by: Jari Kolehmainen <jari.kolehmainen@gmail.com>

### CODE REVIEW

File: extensions/example-extension/webpack.config.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ module.exports = [
     },
     externals: [
       {
-        "@lens/ui-extensions": "var global.LensExtensions",
+        "@k8slens/renderer-extensions": "var global.LensExtensions",
```
Comment (jim-docker) [inline extensions/example-extension/webpack.config.js]: shouldn't `LensExtensions` -> `LensRendererExtensions`?

File: src/extensions/npm/main-extensions/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@k8slens/extensions",
```
Comment (jim-docker) [inline src/extensions/npm/main-extensions/package.json]:
```suggestion
  "name": "@k8slens/main-extensions",
```

File: src/extensions/npm/renderer-extensions/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@k8slens/ui-extensions",
```
Comment (jim-docker) [inline src/extensions/npm/renderer-extensions/package.json]:
```suggestion
  "name": "@k8slens/renderer-extensions",
```

**Review by jim-docker — APPROVED** [review]

File: extensions/example-extension/webpack.config.js (hunk 1 above)
Comment (jakolehm) [reply extensions/example-extension/webpack.config.js]: Yes, good catch!

**Pull request merged**